app.js

```diff
@@ -202,6 +205,54 @@ module.exports = ({ sharedPath: sharedPathIn, port, maxUploadSize, zipCompressio
     });
   }));
 
+
+  // zip file name function 
+  const zipName = () => new Date(Date.now() + 19800000).toISOString().slice(0, -5) + ".zip"
```

Comment:
can you explain this with a comment?

Reply:
This function generates a timestamp-based name for a .zip file